web_view_leaflet_map/static/src/js/view/map/map_renderer.js

```diff
@@ -0,0 +1,31 @@
+odoo.define('web_view_leaflet_map.MapRenderer', function (require) {
+    'use strict';
+
+    var AbstractRenderer = require('web.AbstractRenderer');
+
+    var MapRenderer = AbstractRenderer.extend({
+        tagName: 'table',
+
+        _render: function () {
+            console.debug("MapRenderer:_render");
+            var $mainDiv = $("<div id='div_map' class='_INTRODUCED_IN_RENDER'>Content</div>");
+            this.$el.append($mainDiv);
```

Comment:
Are you sure $el has already been added to the document at this time? If not, document.getElementById will not find it. What if you use container = $mainDiv[0] instead?

Reply:
Thanks ! It worked like a charm. 

I have other problems. (see updated main comment), but PoC is OK !